docs/dictionary-and-concepts/dictionary-charges.md

```diff
@@ -10,32 +10,42 @@ The terms listed here are all terms relating to the topics of defining charges.
 |----------|-------------|
 | **A** |   |
 | **B** |   |
+| Business Reason Code | Indicates the business context, e.g. D10 = 'Request update charge information', which covers all three charge operation types; 'Create', 'Update' and 'Stop' |
 | **C** |   |
-| Charge Name  | The identifyer of a specific charge  |
-| Charge Occurrence | An indication on how often the charge is being billed  |
-| Charge Owner | An identifier for the charge owner  |
-| Charge Type | This could be either a Tariff, Fee or Subscription  |
-| Currency Unit | Defining which currency is used  |
+| Charge Description |  |
+| Charge Id | A charge identifier provided by the Market Participant. Combined with Charge Owner it becomes unique |
+| Charge Name |  |
+| Charge Owner | A charge owner identification, e.g. the Market Participant's GLN or EIC number |
+| Charge Type | The type of charge; tariff, fee or subscription  |
+| Currency | The charge price currency |
 | **D** |   |
 | **E** |   |
 | **F** |   |
 | **G** |   |
 | **H** |   |
 | **I** |   |
+| Industry Classification | Indicates the industry context, e.g. E23 = Electricity |
 | **J** |   |
 | **K** |   |
 | **L** |   |
 | **M** |   |
 | **N** |   |
 | **O** |   |
+| Operation Type | Indicates the type of charge operation, e.g. 'Create', 'Update' or 'Stop' |
 | **P** |   |
-| Price | Charge price  |
+| Position | Indicates a price's position in a charge price list |
+| Price | A charge price |
 | **Q** |   |
-| Quantity | Quantity of a specific type of energy  |
 | **R** |   |
+| Resolution | The resolution of a charge price list, e.g. 15 mins, hourly, daily, monthly |
 | **S** |   |
 | **T** |   |
+| Tax Indicator | Government tax indicator |
+| Time | A point in time where the charge price applies |
```

Comment:
Date or Time? A charges document model has a Request*Date* and a Created*DateTime* and a point model has a *Time*. All are NodaTime Instants. How do we decide when to use which?

Reply:
Please feel free to come up with a suggestion. This was taken from the TimeSeries' ObservationTime, which does not seem fit either here.

Reply:
I think this is a candidate for alignment across domains. You probably don't need to do anything in this PR.

@Sondergaard, is this something you can/will handle, or where should we direct this issue?